### PR TITLE
[FIX] theme_*: default theme improvements

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This PR updates themes to match improvements made to the default theme.
- https://github.com/odoo/odoo/pull/120302

task-3097005

### CODE REVIEW

File: theme_anelusia/views/snippets/s_product_list.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
     </xpath>
     <!-- Item #4 - Img -->
     <xpath expr="(//img)[4]" position="attributes">
-        <attribute name="src">/web_editor/image_shape/website.s_product_list_default_image_4/web_editor/geometric/geo_square_1g</attribute>
```
Comment (qsm-odoo) [inline theme_anelusia/views/snippets/s_product_list.xml:39]: Is not this line a pure unrelated bug fix? If that's the case, you should target the right stable version in another PR with a dedicated commit :+1:

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

File: theme_anelusia/views/snippets/s_product_list.xml (hunk 1 above)
Comment (Brieuc-brd) [reply theme_anelusia/views/snippets/s_product_list.xml:39]: Hey @qsm-odoo 👋 Indeed ! I created a new PR for that, thanks !
https://github.com/odoo/design-themes/pull/689

**Review by qsm-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Comment by qsm-odoo** [comment]
@robodoo retry
